README.md

```diff
@@ -14,30 +14,36 @@ npm i @nypl/nypl-streams-client --save
 
 ```js
 const NyplStreamsClient = require('@nypl/nypl-Streams-client')
-var streamsClient = new NyplStreamsClient({ nyplDataApiClientBase: 'http://example.com/api/v0.1/' })
+const streamsClient = new NyplStreamsClient({ nyplDataApiClientBase: 'https://example.com/api/v0.1/' })
 ```
 
 See [docs/usage.md](docs/usage.md) for complete documentation of Client methods and use.
 
 ### Example 1: Writing data to a stream
 
 To write a single record to a stream (encoded to "MyStream" schema):
+
 ```js
-streamsClient.write('MyStream', { id: 'id1', field1: 1, field2: 2 }).then((resp) => {
-  console.log('Finished writing to stream ' + resp.Records.length)
-}).catch((e) => console.error('Error writing to stream: ', e))
+streamsClient.write('MyStream', { id: 'id1', field1: 1, field2: 2 })
+  .then((resp) => {
```

Comment:
Can you convert this to try/catch for consistency?

Reply:
Updated